src/Application.vala

```diff
@@ -201,7 +201,11 @@ public class Terminal.Application : Gtk.Application {
         );
 
         var new_window_action = new SimpleAction ("new-window", null);
-        new_window_action.activate.connect (new_window);
+        new_window_action.activate.connect (() => {
+            var win = new MainWindow (this, false); // Do not restore tabs
```

Comment:
this will break having the first window be created by the action when we enable DBus activation (the only thing that was stopping us of doing that was the duplicate tab). so the `active_window == null` is necessary here.

Reply:
Not sure what this means :disappointed: as I am not sure how DBus activation works.  Could you give more details or suggested code?